server/protocols/webadmin/webadmin-core/src/main/java/org/apache/james/webadmin/dto/HealthCheckDto.java

```diff
@@ -0,0 +1,25 @@
+package org.apache.james.webadmin.dto;
```

Comment:
You are missing the compulsory Apache V 2.0 header

Reply:
Will add the same.